### PR TITLE
Update Backoffice - Bulk Close WorkOrders

### PR DESCRIPTION
**Pull request by LBHCallumM**
[Ticket](https://hackney.atlassian.net/browse/MR-682?atlOrigin=eyJpIjoiYTA4MDY3NDMwOGM4NDRiMzhhOGQ4ZmFhZjEyMzc5OTIiLCJwIjoiaiJ9)

## Summary of Changes

- Move logic to the utils file for easier testing
- Add better validation to workOrderReferences textField
- Add validation to ensure the closed date is in past
- Added 'clearForm' method to reset form after the user clicks 'Close more WorkOrders' button
- Add more testing
- Add `loginWithDataAdminRole` cypress command

### CODE REVIEW

File: .env.defaults [hunk 1]
```diff
@@ -41,6 +42,16 @@ CYPRESS_PAGE_SIZE_CONTRACTORS=10
 PAGE_SIZE_AGENTS=10
 CYPRESS_PAGE_SIZE_AGENTS=10
 
+### Cypress auth tokens
```
Comment (Asura5) [inline .env.defaults:45]: Maybe a comment to the effect that "these will require populating in .env.local" or something along those help-a-future-developer-out kind of problems

File: src/components/BackOffice/CloseWorkOrders/utils.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+export const formatWorkOrderReferences = (workOrderReferences) => {
+  return workOrderReferences
+    .split('\n')
+    .map((x) => x.trim())
+    .filter((x) => x)
```
Comment (Asura5) [inline src/components/BackOffice/CloseWorkOrders/utils.js:5]: is this intended to remove duplicates. if so, have you tested that this is the case?

File: src/components/BackOffice/CloseWorkOrders/utils.test.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    const pastDate = new Date()
+    pastDate.setDate(pastDate.getDate() - 1)
+
+    // Act
+    const response = dateIsInFuture(pastDate)
+
+    // Assert
+    expect(response).toBe(false)
+  })
+})
```
Comment (Asura5) [inline src/components/BackOffice/CloseWorkOrders/utils.test.js:77]: Add test for duplicate workorders

File: src/components/BackOffice/CloseWorkOrders/utils.js (hunk 2 above)
Comment (LBHCallumM) [reply src/components/BackOffice/CloseWorkOrders/utils.js:5]: It doesn't remove duplicates, but that's definitely worth adding . This takes the contents from the textArea, and returns an array of workOrderReferences. The filter removes nulls

File: cypress/integration/backoffice/close-workorders.spec.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
+/// <reference types="cypress" />
+
+import 'cypress-audit/commands'
+
+describe('Close-WorkOrders', () => {
+  it('Shows access denied when user doesnt have correct permissions', () => {
+    cy.loginWithOperativeRole()
```
[the remark this answers is not in the document]
Comment (LBHCallumM) [reply cypress/integration/backoffice/close-workorders.spec.js:7]: I did this because there is a single test that uses different credentials. Ill see what I can do

File: .env.defaults (hunk 1 above)
Comment (LBHCallumM) [reply .env.defaults:46]: I have added some context

**Review by Asura5 — APPROVED** [review]
